CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Add a flexible http client providing both contract, and only emulating what's necessary
 - HTTP Client Router: route requests to underlying clients
+- TODO for existing clients to become final in 2.0
```

Comment:
i don't think this needs to go into the changelog. the changelog should only contain things people need to act on. so maybe: "Deprecated extending client classes. They will be made final in client-common 2.0"